src/strategies/flex_window.py

```diff
@@ -84,170 +86,321 @@ def step(self, event_list=[]):
                 if type(event) == events.GridOperatorSignal:
                     # update GC info
                     cur_max_power = event.max_power or cur_max_power
-                    cur_window = event.window or cur_window
+                    cur_window = event.window
```

Comment:
I think this hints at a bigger problem - not every GridSignal should have "window" set (e.g. price signals) -> change events.py l. 146 to `('window', bool, None)`, change flex_window l.89 to `if event.window is not None: cur_window = event.window`